L1TriggerConfig/Utilities/src/L1MenuWriter.cc

```diff
@@ -26,6 +27,28 @@ class L1MenuWriter : public edm::EDAnalyzer {
 void L1MenuWriter::analyze(const edm::Event& iEvent, const edm::EventSetup& evSetup){
 
     edm::ESHandle<L1TUtmTriggerMenu> handle1;
+    evSetup.get<L1TUtmTriggerMenuRcd>().get( handle1 ) ;
+    boost::shared_ptr<L1TUtmTriggerMenu> ptr1(new L1TUtmTriggerMenu(*(handle1.product ())));
+
+    edm::Service<cond::service::PoolDBOutputService> poolDb;
+    if( poolDb.isAvailable() ){
+        cond::Time_t firstSinceTime = poolDb->beginOfTime();
+        poolDb->writeOne(ptr1.get(),firstSinceTime,"L1TUtmTriggerMenuRcd");
+    }
+
+}
+
+class L1MenuWriter_ : public edm::EDAnalyzer {
```

Comment:
The only difference between the two versions is the record name: L1TUtmTriggerMenuRcd vs. L1TUtmTriggerMenuO2ORcd. The first one is used for manual O2Oing the menu from static XML and is generated with the L1Trigger/L1TGlobal/python/TriggerMenu_cff in offline environment. The second one is produced by the real O2O job running from the online environment. So there is no old/new relationship between the two classes: I use both of them.

I admit that these two classes could have been merged into a single configurable class. Moreover, the L1Menu is a special case among all of the trigger payloads where we could really get away with using just one record name. Nonetheless, for the moment I've chosen the least intrusive way to extend my existing code for handling the two cases.

The important remark here is that the whole L1TriggerConfig/Utilities package is mostly used for the debugging purposes and was never intended to be polished. So I'd prefer sticking with the current naming (on a promise to modify it in my next PR) rather than updating the branch and making the PR stale for another month waiting for the sign-offs after the break.

P.S. In my next PR I'll also change the L1TriggerConfig/Utilities/src/L1TCaloParamsWriter.cc where I use single and double underscores in the class names for the very same reason.